### ISSUE TITLE
No double opt-in for supporting

### ISSUE THREAD

**Issue by garak** [greydnls/code-manifesto#82]
I just signed to support Code manifesto and I was immediately displayed on the list.
This is not fair, since it's open to possibility to sign other people.
Possible solutions:
* enforce a confirmation via email. Without confirmation, person is not displayed. Slow, but more secure
* send a confirmation afterwards. Person is displayed immediately, but then an email is sent to ensure that they are aware of that. Faster, but less secure (also, it requires a system to delete false subscriptions)

**Comment by metaturso** [comment]
I second this, and my email requests to being removed are being disregarded.

**Comment by Potherca** [comment]
Since the (Laravel based) application has been taken offline and the list of supporters is no longer displayed.

My suggestion for re-establishing the list (if so desired) would be the have a static document and let people add themselves via MR/PRs. (But that's just my 2 cents).

Regardless, I think this issue can be closed now?

**Comment by garak** [comment]
I disagree about closing this issue. The problem is still present inside this repository code, regardless related functionality is online or not.

**Comment by metaturso** [comment]
Any chances of somebody taking action on this soon? The website is back online and still showing outdated and information.
